### PR TITLE
doc: prepare new release 3.8.1

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#4604

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+- HTML code is displayed as text in the description of join space screen and the admin space screens
+- Better styling in the public profile page (some margin at the bottom, add missing home icon, handle long name/username)
+- Update pushpin's repository url
+- Update preview-generator version
+- Fix errors with pushpin version 1.30
+- The color(s) of the sidebar in color.json has been configured
+- Terms and conditions validation are displayed when a user signs in for the first time
+- The space advanced parameters popup opens faster when an image is included in the space description
+- The gallery button in the content list page is only displayed when it is enabled
+- Documentation was added about the usage of env_settings.md and branding.md
```
Comment (gbonaspetti) [inline changelog.md:15]: For me, some of these items are bug fixes. A suggestion:
```suggestion
- The color of the sidebar in color.json can be configured
- Terms and conditions validation are displayed when a user signs in for the first time
- Documentation was added about the usage of env_settings.md and branding.md
- A user can now create a public account
- German translation is available
- The user account interface uses explicit placeholders and labels for guidance
```
For the points that I deleted, if I don't put in the breaking changes, they are bug fixes for me (so add in the "Fixed Issues" section.

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+Docker: [#4603](https://github.com/tracim/tracim/issues/4603)
+Documentation: [#4619](https://github.com/tracim/tracim/issues/4619)
+
+
+### Breaking/Important change
+
+- A user can now create a public account
+- German translation is available
+- The user account interface uses explicit placeholders and labels for guidance
+- An alternative target language can be chosen for content and comment translation
```
Comment (gbonaspetti) [inline changelog.md:45]:
```suggestion
- An alternative target language can be chosen for content and comment translation
- Update pushpin's version
- Update preview-generator version
```

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
+# 3.8.1 / 2021-05-12
+
+### New Features
+
+- Recent activities display faster
+- HTML code is displayed as text in the description of join space screen and the admin space screens
```
Comment (raphj) [inline changelog.md]:
```suggestion
- HTML code is displayed as text in the description of join space and the admin space screens
```

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+- Better styling in the public profile page (some margin at the bottom, add missing home icon, handle long name/username)
```
Comment (raphj) [inline changelog.md]:
```suggestion
- Better styling in the public profile page (some margins at the bottom, add missing home icon, handle long names/usernames)
```

or

```suggestion
- Some polish in the public profile page (some margins at the bottom, add missing home icon, handle long names/usernames)
```

Maybe it is not necessary to enter the details (?)

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,49 @@
...
+- Update pushpin's repository url
```
Comment (raphj) [inline changelog.md]: This is not an issue, we can keep this in the fixed issues only I think

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,43 @@
+# 3.8.1 / 2021-05-12
+
+### New Features
+
+- The color of the sidebar in color.json can be configured
```
Comment (raphj) [inline changelog.md:5]:
```suggestion
- The color of the sidebar can be configured in color.json
```

File: changelog.md [hunk 7]
```diff
@@ -1,3 +1,43 @@
...
+- Terms and conditions validation are displayed when a user signs in for the first time
```
Comment (raphj) [inline changelog.md:6]:
```suggestion
- Terms and conditions are displayed when a user signs in for the first time
```

File: changelog.md [hunk 8]
```diff
@@ -1,3 +1,43 @@
...
+- Documentation was added about the usage of env_settings.md and branding.md
```
Comment (raphj) [inline changelog.md:7]:
```suggestion
- env_settings.md and branding.md have been documented
```

File: changelog.md [hunk 9]
```diff
@@ -1,3 +1,43 @@
...
+- A user can now create a public account
+- German translation is available
```
Comment (raphj) [inline changelog.md:9]:
```suggestion
- The user interface is now available in German
```

**Review by raphj — APPROVED** [review]

File: changelog.md [hunk 10]
```diff
@@ -1,3 +1,41 @@
...
+[#4591](https://github.com/tracim/tracim/issues/4591)
+[#4521](https://github.com/tracim/tracim/issues/4521)
+[#3899](https://github.com/tracim/tracim/issues/3899)
+Frontend/Backend: [#4590](https://github.com/tracim/tracim/issues/4590)
+[#4539](https://github.com/tracim/tracim/issues/4539)
+[#4557](https://github.com/tracim/tracim/issues/4557)
+[#4586](https://github.com/tracim/tracim/issues/4586)
+Backend: [#4587](https://github.com/tracim/tracim/issues/4587)
+[#3911](https://github.com/tracim/tracim/issues/3911)
+[#4632](https://github.com/tracim/tracim/issues/4632)
```
Comment (grignards) [inline changelog.md:26]:
```suggestion
[#4632](https://github.com/tracim/tracim/issues/4632)
[#4631](https://github.com/tracim/tracim/issues/4631)
```

**Pull request merged**